src/main/java/com/bobocode/dto/ErrorTO.java

```diff
@@ -0,0 +1,5 @@
+package com.bobocode.dto;
+
+public record ErrorTO(String errorMessage) {
```

Comment:
@kebann I suppose TO means Transfer Object, but I've never seen such naming before. So personally **I would stick to `DTO` which is a known pattern.**

Reply:
Will change to `DTO`
I've seen the `TO` prefix used on a couple of projects.